ArcGISRuntimeSDKQt_QMLSamples/Scenes/FilterFeaturesInScene/README.md

```diff
@@ -0,0 +1,41 @@
+# Filter features in scene
+
+Filter 3D scene features out of a given geometry with a polygon filter.
+
+![](screenshot.png)
+
+## Use case
+
+You can selectively show or hide features within a scene view using `setFeaturesVisible` to filter specific features or set a `SceneLayerPolygonFilter` to filter features given a polygon. Through these methods, you can directly control what users see within a specific scene view to give a more focused or cleaner user experience.
+
+## How to use the sample
+
+The sample initializes showing the 3D buildings OpenStreetMap layer. Click the "Load detailed buildings" button to load an additional scene layer that contains more detailed buildings. Notice how the two scene layers overlap and clip into each other. Click the "Filter OSM buildings in extent" button, to set a `SceneLayerPolygonFilter` and filter out the OpenStreetMap buildings within the extent of the detailed buildings scene. Notice how the OSM buildings within and intersecting the extent of the detailed buildings layer are hidden. Click the "Reset scene" button to hide the detailed buildings scene layer and clear the OSM buildings filter.
+
+## How it works
+
+1. Construct a `Basemap` for the scene using an OpenStreetMap Topographic `VectorTileLayer` and the OpenStreetMap 3D Buildings `ArcGISSceneLayer` as baselayers.
```

Comment:
if we put it in backticks, we should use the full proper name. If we want to be casual, can just remove the backticks
```suggestion
1. Construct a `Basemap` for the scene using an OpenStreetMap Topographic `ArcGISVectorTileLayer` and the OpenStreetMap 3D Buildings `ArcGISSceneLayer` as baselayers.
```